app/assets/templates/views/pick_scorer.html

```diff
@@ -3,6 +3,9 @@
   <h3 class="modal-title">How would you like to score this case?</h3>
 </div>
 <div class="modal-body">
+  <div ng-if="!scorerAccessible" class="alert alert-warning" role="alert">
+    The scorer {{activeScorer.name}} used by this case is NOT shared via a team with you, so if you switch away from it you won't have access to it again.
```

Comment:
If I have access to a case that uses a scorer, and I want to see a different scorer's output, I surely ought to be able to return the case to the state that I found it, putting back the previous scorer, whether or not anyone has shared it.

Reply:
i think that is kind of the question, how do you get access...  Right now, the only way to "get back access" would be to a) make you the owner, or b) Add the scorer to a team you are in...    We don't have any history type tracking that would let us pluck it out of the history.    The warning at least lets you know so you could ping the owner to share it with you?

Reply:
i think the same pattern will apply if you have a try with a search endpoint, you can see it.  but if you switch it to one you either own or is in your team, well then you can't get it back....

Reply:
Disagree. If I can see it, I can return to it. 

When I load up the case and view it, the case should record the displayed scorer, so that I can return to that state. More generally, each try should know not just its results, but the scorer at the time of the try. If I change scorers, I get a new try. Makes returning to alternate states easier (no queries were run to bring you this message...).

Reply:
Humm...  that's an interesting point..   if you change score, you change try?   because then we *would* have the history.........   If we want to do that, let's go ahead and make a NEW ticket for just that feature.   Something like "YOu change anything, like a scorer, you get a new try"..  cause right now, I don't actually know what triggers the new try OTHER than rerunning the queries".   So, if you change the scorer, we just create a new try with that scorer...